deep_qa/layers/wrappers/fixed_time_distributed.py

```diff
@@ -0,0 +1,37 @@
+from keras import backend as K
+from keras.layers import TimeDistributed
+
+class FixedTimeDistributed(TimeDistributed):
```

Comment:
I think we should call this `TimeDistributed`, so that if/when Keras fixes its class, we just have to change an import in the model code (does that make sense?).  I've been meaning to do this myself, but haven't yet.  So can you do the import above as `from keras.layers import TimeDistributed as KerasTimeDistributed`, and call this class `TimeDistributed`?  This is similar to how we did the [`Highway`](https://github.com/allenai/deep_qa/blob/47f38139f31b216c00762d28f1786d65dd68a9ee/deep_qa/layers/highway.py) layer.  And then you'd want to change the name of the file, too.